pkg/kn/commands/wait_flags.go

```diff
@@ -50,3 +52,10 @@ func (p *WaitFlags) AddConditionWaitFlags(command *cobra.Command, waitTimeoutDef
 	timeoutUsage := fmt.Sprintf("Seconds to wait before giving up on waiting for %s to be %s.", what, until)
 	command.Flags().IntVar(&p.TimeoutInSeconds, "wait-timeout", waitTimeoutDefault, timeoutUsage)
 }
+
+// AddErrorWindow will add a flag to configure the error window, that is,
+// the time to wait between intermediate false ready conditions
+func (p *WaitFlags) AddErrorWindow(command *cobra.Command, what, until string) {
```

Comment:
IMHO it doesn't need to be a standalone add functions. It seems to me like a `wait` family option.